design-patterns/gc-institution/index.json-ld

```diff
@@ -0,0 +1,157 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "About the institution pattern",
+		"fr": "À propos de l’institution"
+	},
+	"description": {
+		"en": "Section on an institutional landing page that provides links to corporate, program and policy information",
+		"fr": "Section d’une page d’accueil institutionnelle offrant des liens vers des renseignements relatifs à un organisme, à un programme ou à une politique en particulier."
+	},
+	"modified": "2024-01-11",
+	"componentName": "gc-institution",
+	"status": "stable",
+	"version": "1.0",
```

Comment:
We will need to use the documentation format that @Garneauma did for design pattern where there is no component stability indication neither a version number associated with it neither versioning api documentation.